apps/next/src/pages/Contacts/ContactList.tsx

```diff
@@ -0,0 +1,83 @@
+import { Contact } from '@avalabs/types';
+import { useHistory } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { Button, Stack, Typography } from '@avalabs/k2-alpine';
+
+import { useContactsContext } from '@core/ui';
+
+import { Page } from '@/components/Page';
+import { getContactsPath } from '@/config/routes';
+
+export const ContactList = () => {
+  const { t } = useTranslation();
+  const { contacts } = useContactsContext();
+
+  return (
+    <Page
+      title={t('Contacts')}
+      description={
+        contacts.length > 0
+          ? t('{{count}} saved addresses', { count: contacts.length })
```

Comment:
just a question in general, we are not dealing with the issue of having only one thing? So here if the user only have one address saved it should read `1 saved address` but right now it reads `1 saved addresses`
\#justAsking